arch/x86/chunkset_avx512.c

```diff
@@ -98,6 +98,10 @@ static inline uint8_t* CHUNKCOPY(uint8_t *out, uint8_t const *from, unsigned len
     return out;
 }
 
+/* MSVC compiler decompression bug when optimizing for size */
+#if defined(_MSC_VER) && _MSC_VER < 1943
+#  pragma optimize("", off)
```

Comment:
Does it still work if the first parameter is `"t"`, as it would not disable optimizations when `/Ot` is enabled.

Reply:
I tried it but it does not work right.

Reply:
`/Ot` optimizes for speed, so the comment is then kind of inaccurate...

Reply:
Don't quite understand your last comment. That optimization string argument only works if you are starting from enabling optimization, not starting from disabling optimization.

Reply:
`/Ot` and `/Os` are mutually exclusive, however both also enable for example `/Og` (common sub-expression elimination). With current patch, it will disable all optimizations and thus degrade performance even if optimizing for speed.